src/components/VueDialogCard.vue

```diff
@@ -0,0 +1,146 @@
+<script lang="ts">
+import { defineComponent, computed } from 'vue';
+
+// types
+import { DialogCard } from 'components/types';
+
+// Component used to render dialog window which you can fill with custom content
```

Comment:
This multi-line comment (for multi-line commenting use `/* *`/ please) should be as component doc header on the first line.

Reply:
Edited component docs.